contracts/scripts/deployments/diamonds/DeploySpace.s.sol

```diff
@@ -189,6 +189,115 @@ contract DeploySpace is DiamondHelper, Deployer {
+        banning = banningHelper.deploy(deployer);
+        addCut(banningHelper.makeCut(banning, IDiamond.FacetCutAction.Add));
+      } else if (
+        facetNameHash == keccak256(abi.encodePacked("MembershipFacet"))
+      ) {
+        membership = membershipHelper.deploy(deployer);
+        addCut(
+          membershipHelper.makeCut(membership, IDiamond.FacetCutAction.Add)
+        );
+      } else if (
+        facetNameHash == keccak256(abi.encodePacked("MembershipMetadata"))
+      ) {
+        membershipMetadata = membershipMetadataHelper.deploy(deployer);
+        addCut(
+          membershipMetadataHelper.makeCut(
+            membershipMetadata,
+            IDiamond.FacetCutAction.Add
+          )
+        );
+      } else if (
+        facetNameHash == keccak256(abi.encodePacked("EntitlementDataQueryable"))
+      ) {
+        entitlementDataQueryable = entitlementDataQueryableHelper.deploy(
+          deployer
+        );
+        addCut(
+          entitlementDataQueryableHelper.makeCut(
+            entitlementDataQueryable,
+            IDiamond.FacetCutAction.Add
+          )
+        );
+      } else if (
+        facetNameHash == keccak256(abi.encodePacked("EntitlementsManager"))
+      ) {
+        entitlements = entitlementsHelper.deploy(deployer);
+        addCut(
+          entitlementsHelper.makeCut(entitlements, IDiamond.FacetCutAction.Add)
+        );
+      } else if (facetNameHash == keccak256(abi.encodePacked("Roles"))) {
+        roles = rolesHelper.deploy(deployer);
+        addCut(rolesHelper.makeCut(roles, IDiamond.FacetCutAction.Add));
+      } else if (facetNameHash == keccak256(abi.encodePacked("Channels"))) {
+        channels = channelsHelper.deploy(deployer);
+        addCut(channelsHelper.makeCut(channels, IDiamond.FacetCutAction.Add));
+      } else if (
+        facetNameHash == keccak256(abi.encodePacked("TokenPausableFacet"))
+      ) {
+        tokenPausable = tokenPausableHelper.deploy(deployer);
+        addCut(
+          tokenPausableHelper.makeCut(
+            tokenPausable,
+            IDiamond.FacetCutAction.Add
+          )
+        );
+      } else if (facetNameHash == keccak256(abi.encodePacked("PrepayFacet"))) {
+        prepay = prepayHelper.deploy(deployer);
+        addCut(prepayHelper.makeCut(prepay, IDiamond.FacetCutAction.Add));
+      } else if (
+        facetNameHash == keccak256(abi.encodePacked("ReferralsFacet"))
+      ) {
+        referrals = referralsHelper.deploy(deployer);
+        addCut(referralsHelper.makeCut(referrals, IDiamond.FacetCutAction.Add));
+      } else if (
+        facetNameHash == keccak256(abi.encodePacked("EntitlementGated"))
+      ) {
+        entitlementGated = entitlementGatedHelper.deploy(deployer);
+        addCut(
+          entitlementGatedHelper.makeCut(
+            entitlementGated,
+            IDiamond.FacetCutAction.Add
+          )
+        );
+      }
+    }
+
+    return
+      Diamond.InitParams({
```

Comment:
don't need to return InitParams

Reply:
👍

Reply:
76e0b6ce6d8744d1185cee3a3e1cc45acad5f03a